### ISSUE TITLE
completed-docs still joins descriptors by commas

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [palantir/tslint#3378]
### Bug Report

- __TSLint version__: 5.8
- __TypeScript version__: 2.6
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
class Foo {
    public async bar() { /* ... */ }
}
```

with `tslint.json` configuration:

```json
{
    "rules": {
        "completed-docs": true
    }
}
```

#### Actual behavior

`bar` should be described as "`public async`".

#### Expected behavior

```
ERROR: 5:2   completed-docs  Documentation must exist for public,async methods.
```

I feel silly for forgetting to `" "` in the `.join`. ☹️

**Comment by ajafff** [comment]
Fixed by #3379
